### ISSUE TITLE
Feature request: mitigate against notes with dozens/hundreds of blank lines

### ISSUE THREAD

**Issue by StevenBlack** [damus-io/damus#84]
William @jb55 this is a screen shot of the following note ID

```
note1vratuvvqevzthn4y755lc55x7232uh0s8smk4ze2j8fhrspyzwksf2ylwn
```

Is this fairly simple to mitigate, or nah?  

There's SPAM, and then there's SPAM that makes the app appear broken because my first impression was, the app's not loading the list of notes.

**Added**: Also wondering, in a usability sense, is this mitigated somewhat by making the list background color a tad different than the note background color?

![IMG_799ABA65F992-1](https://user-images.githubusercontent.com/80144/208972654-7517772d-5b7d-4791-a87b-bb992a2bb826.jpeg)

**Comment by jb55** [comment]
we should collapse large posts

**Comment by alltheseas** [comment]
> we should collapse large posts

In 1.5 long notes with text/characters. are abbreviated with "show more". 

Blank text issues remains open. Adding to backlog.

**Comment by alltheseas** [comment]
Appears resolved 1.6
